test/test-scratch.js

```diff
@@ -92,6 +92,7 @@ it('scratch', async () => {
     el.shadowRoot.querySelector('span').textContent === '',
     'should update the DOM bindings'
   );
+  // eslint-disable-next-line require-atomic-updates
```

Comment:
I'll be honest, I don't fully understand why eslint was complaining here. However, it's in test code and I'm not too worried about adding this line.

Reply:
is it a false positive? https://github.com/eslint/eslint/issues/11899

Reply:
Ah, great find. I'll add that to the comment here. Yes, I do think this is a false positive.